daml-lf/scenario-interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/ScenarioRunner.scala

```diff
@@ -176,6 +176,7 @@ final case class ScenarioRunner(
       cbMissing: Unit => Boolean,
       cbPresent: ContractInst[Tx.Value[ContractId]] => Unit) = {
 
+    // Once we support multi-party reads, we can drop this restriction.
```

Comment:
Should we attach a ticket to this so we don't forget it?

Reply:
You can’t implement multi-party submissions without this so I don’t think there is any danger of forgetting this.